include/bluetooth/l2cap.h

```diff
@@ -150,6 +150,22 @@ struct bt_l2cap_chan {
 #endif /* CONFIG_BT_L2CAP_DYNAMIC_CHANNEL */
 };
 
+/** @brief Get @c chan->state.
+ *
+ * Returns #BT_L2CAP_CONNECTED if @kconfig{CONFIG_BT_L2CAP_DYNAMIC_CHANNEL} is
+ * disabled.
+ */
+static inline bt_l2cap_chan_state_t bt_l2cap_chan_get_state(
+	struct bt_l2cap_chan *chan)
+{
+#if defined(CONFIG_BT_L2CAP_DYNAMIC_CHANNEL)
+	return chan->state;
```

Comment:
Is it so that `chan->state` doesn't exist if `L2CAP_DYNAMIC_CHANNEL` isn't defined? If it does always exist, then please use `if (IS_ENABLED(CONFIG_BT_L2CAP_DYNAMIC_CHANNEL)` instead.

Reply:
Yes. The field `state` is not part of the struct when `L2CAP_DYNAMIC_CHANNEL` is not defined. I'll reword the doc to make it clearer.